packages/integration-tests/index.js

```diff
@@ -676,20 +680,23 @@ async function buildWorkflow(
   meta
 ) {
   setProcessEnvironment(stackName, bucketName);
+
   const template = await getWorkflowTemplate(stackName, bucketName, workflowName);
-  let collectionInfo = {};
-  if (collection) {
-    collectionInfo = await new Collection()
-      .get({ name: collection.name, version: collection.version });
-  }
-  let providerInfo = {};
-  if (provider) {
-    providerInfo = await new Provider().get({ id: provider.id });
-  }
+  const collectionInfo = collection
+    ? await new Collection().get({
```

Comment:
Style:   I tend to not like multi-line ternaries, particularly for readability reasons, but that's my opinion.  I don't know that I've heard the team have a discussion on it, and we don't have lint rules enforcing either/or..... probably worth a quick conversation but nothing to hold up the PR. 

That said, this particular use of it was pretty difficult to read, given the multi-line split out of the Collection().get call, could we do something like: 

const collectionInfo = collection
  if (collection) {	    ? await new Collection().get({
 

 
Style:   I tend to not like multi-line ternaries, particularly for readability reasons, but that's a personal opinion.  I don't know that I've heard the team have a discussion on it, and we don't have lint rules enforcing either/or..... probably worth a conversation at some point, but nothing to hold up the PR.    

That said, this particular use of it felt somewhat awkward to read at speed given the multi-line split out of the Collection().get call,  doubly so since that's not something I think is consistently done in the codebase - if this has to be a multi-line ternary, could we do something like: 

```
const collectionInfo = collection 
  ? await new Collection().get({name: collection.name, version: collection.version })
  : {};
```

or if 80 cols is a goal here, alternately 

``` 
let collectionInfo = {};
if (collection){
collectionInfo = await new Collection().get({
name: collection.name, version: collection.version
});
}
```
?

Reply:
Agreed. The wrapping of this ternary expression is not ideal.

As you suspected, I avoided wrapping shown in your first suggestion in order to avoid exceeding 80 columns. (Aside: my preference is to limit code to 80 columns, unless impossible for some reason, if for no other reason than to enable readability in side-by-side code diffs, such as visible in GitHub, or some other tool.)

Regarding your second suggestion, I'm not a fan of using `let` when using `const` is possible. Of course, we're now having to determine which of 2 ugly alternatives to use: either the ugly wrapping, or the ugly use of `let`.

However, I do have another suggestion that avoids `let` and also avoids the hard-to-read wrapping. How about this instead?
```
  const { name, version } = collection || {};
  const collectionInfo = collection
    ? await new Collection().get({ name, version })
    : {};
 ```

Reply:
Yeah, that's excellent, it neatly addresses the concern.